docs/introduction.md

```diff
@@ -2,9 +2,109 @@
 
 # Dela
 
-Dela stands for Dedis Ledger Architecture. The goal of the framework is to
-provide a set of modules that will work together to run a distributed ledger, or
-only a part of it.
+Dela stands for DEDIS Ledger Architecture. It is both a set of abstractions and
+an implementation of a distributed ledger architecture.
+
+Dela has 2 main purposes:
+
+- Provide a modular, global-purpose, and universal framework that describes a
+  minimal and extended set of abstractions for a distributed ledger
+  architecture.
+- Provide multiple modules implementations that can be combined to run a
+  distributed ledger.
+
+With Dela you can:
+
+- Learn the architecture of a distributed ledger
+- Run your blockchain / distributed ledger
+- Implement and test your new idea that will revolutionize the blockchain world
+  by adding your new module's implementation to the Dela ecosystem
+
+## Distributed ledger
+
+A distributed ledger aims to solve the following problem: how can a set of
+entities agree on a state without any central entity? In other words: how can a
+group of computers (or *nodes*) cooperate to maintain a database? This problem
```

Comment:
```suggestion
group of computers (or *nodes*) cooperates to maintain a database? This problem
```

Reply:
Actually this is right since I consider the group as a whole (https://brians.wsu.edu/2016/05/19/group-plural-vs-singular/)

Reply:
Oh interesting !